### PR TITLE
add Chrome globals

### PR DESCRIPTION
**Pull request by sindresorhus**
I got all the globals in Chrome, filtered away the `builtin` ones, and added the new ones to the list.

I'm scared by how many globals there actually are...

@nzakas Do we want all these?

-

// @es128 @jrvidal

### CODE REVIEW

**Comment by es128** [comment]
:+1: I guess. 

I don't suppose it's possible, but it would be nice if there was a way to distinguish between what people actually use in their web apps vs just what the browser happens to make available.

**Comment by es128** [comment]
I guess what we are seeing with the gradual trickling in of PRs to add to this list a bit at a time is that anything made available is liable to be used by someone eventually. 

So yeah, :+1:, not as half-hearted this time

**Comment by sindresorhus** [comment]
Yup

File: globals.json [hunk 1]
```diff
@@ -83,52 +110,106 @@
...
+		"CSSSupportsRule": false,
+		"CSSUnknownRule": false,
+		"CSSViewportRule": false,
 		"CustomEvent": false,
+		"DataTransfer": false,
+		"DataTransferItem": false,
+		"DataTransferItemList": false,
 		"Debug": false,
 		"defaultStatus": false,
+		"defaultstatus": false,
```
Comment (nzakas) [inline globals.json]: Is this correct? Capitalized?

Comment (arthurvr) [inline globals.json]: Seems to be correct. I have that global in my chrome at least.

File: globals.json [hunk 2]
```diff
@@ -220,42 +355,86 @@
...
 		"removeEventListener": false,
+		"Request": false,
 		"requestAnimationFrame": false,
 		"resizeBy": false,
 		"resizeTo": false,
+		"Response": false,
+		"RTCIceCandidate": false,
+		"RTCSessionDescription": false,
 		"screen": false,
+		"Screen": false,
```
Comment (nzakas) [inline globals.json]: Duplicate

Comment (arthurvr) [inline globals.json]: Notice the capital `S`.

add_chrome_globals_by_sindresorhus_ _pull_request__41_ _sindresorhus_globals

File: globals.json [hunk 3]
```diff
@@ -431,20 +631,50 @@
 		"WebGLShaderPrecisionFormat": false,
 		"WebGLTexture": false,
 		"WebGLUniformLocation": false,
+		"WebKitAnimationEvent": false,
```
Comment (nzakas) [inline globals.json]: Not sure about these. We have stayed away from browser specific globals so far.

File: globals.json [hunk 4]
```diff
@@ -431,20 +631,50 @@
...
+		"XMLHttpRequestUpload": false,
 		"XMLSerializer": false,
 		"XPathEvaluator": false,
 		"XPathException": false,
 		"XPathExpression": false,
 		"XPathNamespace": false,
 		"XPathNSResolver": false,
-		"XPathResult": false
+		"XPathResult": false,
+		"XPathResult": false,
```
Comment (nzakas) [inline globals.json]: Duplicate?

File: globals.json (hunk 3 above)
Comment (sindresorhus) [inline globals.json]: Fixed

File: globals.json (hunk 4 above)
Comment (sindresorhus) [inline globals.json]: Fixed

**Comment by sindresorhus** [comment]
@nzakas Everything resolved. Anything else?

**Comment by nzakas** [comment]
Lgtm :+1:

**Pull request merged**